### PR TITLE
feat(home): update home content

### PR DESCRIPTION
**Pull request by ami**
｡ﾟ✶ฺ.ヽ(*´∀`*)ﾉ.✶ฺﾟ｡

### CODE REVIEW

File: app/components/home.jsx [hunk 1]
```diff
@@ -41,14 +41,15 @@ export class Home extends Component {
...
-              <h2>A tool for tracking your <a href="http://bulbapedia.bulbagarden.net/wiki/Living_Pok%C3%A9dex" target="_blank" className="link">Living Dex</a> progress. Easily toggle between and track your captured Pokémon, find the locations of those left to be captured, and share a public link with others to see how you can help each other out. Check out an example living dex <Link className="link" to="/u/ashketchum10/living-dex">here</Link>!</h2>
+              <h2>A tool for tracking your Living Dex progress for <Link className="link" alt="Sample Sun & Moon Living Dex" to="/u/ashketchum10/alola-living-dex">Gen 7 (Pokémon Sun & Moon)</Link> or <Link className="link" alt="Sample Gen 6 Living Dex"  to="/u/ashketchum10/gen-6-living-dex">Gen 6</Link>. Easily toggle between and track your captured Pokémon, find the locations of those left to be captured, and share a public link with others to see how you can help each other out. Manage all your dexes on one <Link className="link" alt="Sample Profile" to="/u/ashketchum10">profile</Link>, and even track a <Link className="link" alt="Sample Shiny Living Dex" to="/u/ashketchum10/shinies">shiny living dex</Link>!</h2>
```
Comment (robinjoseph08) [inline app/components/home.jsx]: - you should replace all `&` with `&amp;`, even those in attributes according to [this guy](http://stackoverflow.com/a/3705601/2581607)
- extra space after `"Sample Gen 6 Living Dex"`

File: app/components/home.jsx [hunk 2]
```diff
@@ -41,14 +41,15 @@ export class Home extends Component {
...
 
-              <p>This project is open source, and you can find the code on Github (<a href="https://github.com/robinjoseph08/pokedextracker.com" target="_blank" className="link">website</a> &amp; <a href="https://github.com/robinjoseph08/api.pokedextracker.com" target="_blank" className="link">API</a>). Feel free to report issues, suggest features, or even submit a pull request!</p>
+              <p>This project is open source, and you can find the code on <a href="https://github.com/pokedextracer/pokedextracker.com" target="_blank" className="link">Github</a>. Feel free to report issues, suggest features, or even submit a pull request!</p>
```
Comment (robinjoseph08) [inline app/components/home.jsx]: - `Github` ➡️ `GitHub`
- trackkkkkkkkkker
- should we link to the repo or the org?

File: app/components/home.jsx [hunk 3]
```diff
@@ -41,14 +41,15 @@ export class Home extends Component {
...
 
               {ctas}
 
               <div className="social">
                 <a href="https://twitter.com/PokedexTracker" target="_blank" className="link"><i className="fa fa-twitter" /></a>
+                <a href="/blog" target="_blank" className="link"><i className="fa fa-rss" /></a>
```
Comment (robinjoseph08) [inline app/components/home.jsx]: can you add the trailing slash? `/blog/`

**Comment by ami** [comment]
@robinjoseph08 upppppdatedddd

File: app/components/home.jsx [hunk 4]
```diff
@@ -41,18 +41,19 @@ export class Home extends Component {
...
               {ctas}
 
               <div className="social">
                 <a href="https://twitter.com/PokedexTracker" target="_blank" className="link"><i className="fa fa-twitter" /></a>
+                <a href="/blog/" target="_blank" className="link"><i className="fa fa-rss" /></a>
               </div>
             </div>
           </div>
-
+, me
```
Comment (robinjoseph08) [inline app/components/home.jsx]: wat

**Pull request merged**